src/app/pages/send/send.component.tsx

```diff
@@ -67,7 +66,7 @@ export class SendPage extends Component<IProps, IState> {
     public render() {
         return (
             <div class="send-page">
-                <LayoutGrid>
+                <LayoutGrid class="no-padding-top-bottom">
```

Comment:
why did you remove the padding here ?

Reply:
This indeed could be a mistake, I did it because the confirm button on the 320 × 568px disappears also when you press the advance button. So I did this which indeed doesnt fixes this entirely. I will add back the padding and maybe think of another sollution for the confirm button for this case also ? Maybe a scrollable screen ?

Reply:
now it seems to be ok ! :)